### PR TITLE
Production configuration for Heavy Ion 2022

### PR DESCRIPTION
**Pull request by germanfgv**
Production configuration for HI test Nov 2022

### CODE REVIEW

File: etc/HIReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -89,33 +106,38 @@
...
+expressProcVersion = dt
+alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "103X_dataRun2_Express_v2"
-promptrecoGlobalTag = "103X_dataRun2_Prompt_v3"
-alcap0GlobalTag = "103X_dataRun2_Prompt_v3"
+expressGlobalTag = "124X_dataRun3_Express_HIreplay2022_v1"
+promptrecoGlobalTag = "124X_dataRun3_Prompt_HIreplay2022_v1"
+alcap0GlobalTag = "124X_dataRun3_Prompt_HIreplay2022_v1"
```
Comment (francescobrivio) [inline etc/HIReplayOfflineConfiguration.py:140]: @germanfgv please use the "prod" GTs here as well:
```suggestion
expressGlobalTag = "124X_dataRun3_Express_v9"
promptrecoGlobalTag = "124X_dataRun3_Prompt_v10"
alcap0GlobalTag = "124X_dataRun3_Prompt_v10"
```

File: etc/HIReplayOfflineConfiguration.py [hunk 2]
```diff
@@ -1318,414 +1465,296 @@
...
-               dqm_sequences = [ "@common" ],
-               scenario = hiTestppScenario)
+               write_miniaod=False,
+               do_reco=True,
+               raw_to_disk=True,
+               write_dqm=True,
+               alca_producers=["SiStripCalZeroBias", "SiStripCalMinBias","TkAlMinBias",
+                               "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
+                               "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
+               dqm_sequences=["@commonSiStripZeroBias"],
```
Comment (francescobrivio) [inline etc/HIReplayOfflineConfiguration.py:1739]: @germanfgv please note that in [1c2f703](https://github.com/dmwm/T0/pull/4778/commits/1c2f703d2e8a1767fbdbec287737e9b828815421) (last commit of my replay) I added some `dqm_sequences` here: if the replay there is successful we should add them here as well.

File: etc/HIProdOfflineConfiguration.py [hunk 3]
```diff
@@ -1429,404 +1529,295 @@
...
-               dqm_sequences = [ "@common", "@ecal", "@egamma" ],
-               scenario = hiTestppScenario)
+               write_miniaod=False,
+               do_reco=True,
+               raw_to_disk=True,
+               write_dqm=True,
+               alca_producers=["SiStripCalZeroBias", "SiStripCalMinBias","TkAlMinBias",
+                               "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
+                               "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
+               dqm_sequences=["@commonSiStripZeroBias"],
```
Comment (francescobrivio) [inline etc/HIProdOfflineConfiguration.py:1802]: same comment here about the `dqm_sequences` added in [1c2f703](https://github.com/dmwm/T0/pull/4778/commits/1c2f703d2e8a1767fbdbec287737e9b828815421)

**Comment by francescobrivio** [comment]
FYI @ebrondol as ORM 😄

File: etc/HIReplayOfflineConfiguration.py (hunk 1 above)
Comment (germanfgv) [reply etc/HIReplayOfflineConfiguration.py]: Done

File: etc/HIReplayOfflineConfiguration.py (hunk 2 above)
Comment (germanfgv) [reply etc/HIReplayOfflineConfiguration.py]: done

File: etc/HIProdOfflineConfiguration.py (hunk 3 above)
Comment (germanfgv) [reply etc/HIProdOfflineConfiguration.py]: done

**Pull request merged**